xbmc/addons/Visualization.cpp

```diff
@@ -152,4 +153,16 @@ void CVisualization::transfer_preset(void* kodiInstance, const char* preset)
   addon->m_presets.emplace_back(preset);
 }
 
+void CVisualization::clear_presets(void* kodiInstance)
+{
+  CVisualization *addon = static_cast<CVisualization*>(kodiInstance);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types